### PR TITLE
Bump 7.3.4

### PR DESCRIPTION
**Pull request by ionelmc**

### CODE REVIEW

**Comment by ionelmc** [comment]
@dhermes @refi64 it's mergeable now.

File: pypy2/README.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
     A Chocolatey package for PyPy.
 
-**Current version:** 7.3.1 (Python 2.7.13)
+**Current version:** 7.3.4 (Python 2.7.13)
```
Comment (dhermes) [inline pypy2/README.rst]: Shouldn't this say 2.7.18?
```suggestion
**Current version:** 7.3.4 (Python 2.7.18)
```

**Comment by ionelmc** [comment]
Fixed now.

**Review by dhermes — APPROVED** [review]
LGTM

**Comment by ionelmc** [comment]
I don't have write access, please merge it.

**Comment by ionelmc** [comment]
@dhermes  67cc017 is missing the 7.3.4 tag (publishing failed cause of that).

**Comment by dhermes** [comment]
SG; I just created https://github.com/refi64/chocolatey-pypy/releases/tag/7.3.4

**Pull request merged**